tests/keras/backend/backend_test.py

```diff
@@ -814,6 +812,7 @@ def test_conv2d(self):
             with pytest.raises(ValueError):
                 k.conv2d(k.variable(xval), k.variable(kernel_val), data_format='channels_middle')
 
+    @pytest.mark.skip
```

Comment:
As discussed please add message on why we skip this.
Also, This should be skipped only if K.backend() == 'mxnet'